routes/nickroutes.js

```diff
@@ -5,50 +5,96 @@ import chalk from 'chalk';
 import fetch from 'node-fetch';
 
 import db from '../database/initializeDB.js';
+import awardMapCustom from '../controllers/awardsController.js';
 
 const router = express.Router();
 
+function getIDByValue(object, value) {
+  return object.filter((item) => item.award_title === value);
+}
+
+function getTableRows(table) {
+  return `SELECT * FROM ${table}`;
+}
+
+
+
 /// /////////////////////////////////
 /// ////awards Endpoints////////
 /// /////////////////////////////////
 router.route('/awards')
-  .get(async (req, res) => {
+  .get(async(req, res) => {
     try {
+      const result = await db.sequelizeDB.query(awardMapCustom, {
+        type: sequelize.QueryTypes.SELECT
+      });
       console.log('touched /awards with GET');
-      res.json({message: 'touched /awards with GET'});
-    } catch (error) {
-      console.log(error);
-      res.json({error: 'Something went wrong on the server'});
+      res.json(result);
+    }
+    catch (error) {
+      console.error(error);
+      res.send('Something went wrong on /awards at GET');
     }
   })
-
-  .put(async (req, res) => {
+  .put(async(req, res) => {
     try {
-      console.log('touched /awards with PUT');
-      res.json({message: 'touched /awards with PUT'});
+      const awards = await db.sequelizeDB.query(getTableRows('awards'), {
+        type: sequelize.QueryTypes.SELECT
+      });
+      const awardStatement = `SELECT * FROM awards WHERE award_title = "${req.body.award_title}"`;
+      const selectedMovie = await db.sequelizeDB.query(awardStatement, {
+        type: sequelize.QueryTypes.SELECT
+      });
+      const awardTitle = getIDByValue(award_title, req.body.award_title);
```

Comment:
This should be a variable named awardName (instead of awardTitle) since you used it as awardName right below it